homeassistant/components/lg_netcast/media_player.py

```diff
@@ -230,6 +236,11 @@ def volume_down(self):
         """Volume down media player."""
         self.send_command(25)
 
+    def set_volume_level(self, volume):
+        """Set volume level, range 0..1."""
+        self._client.set_volume(float(volume * 100))
+        self.__update_volume()
```

Comment:
Why do we force an update here, while not on other changes?
Additionally, it seems that the update method is throttled, this would bypass that, isn't that an issue?

Reply:
> Additionally, it seems that the update method is throttled, this would bypass that, isn't that an issue?

By the way, I don’t know why there is throttling in the update function. This breaks the manual update service (`homeassistant.update_entity`), without it everything works great and responds faster. I think we can remove it?

Reply:
@R0nd I think we can suppress `self.__update_volume()` in this method now and improve the state updates in the next PR.